### ISSUE TITLE
Spotter "modal" issues

### ISSUE THREAD

**Issue by guillep** [pharo-spec/NewTools#187]
Spotter "modalness" is strange.
 - I cannot finish understanding the gray background (which is I think a general issue of all modals). To me it is a bit disrupting, even more because of the low contrast with the new spotter window. Spotter is now full gray, the modal background is gray too! It is strange, my eyes do not go to spotter directly

![image](https://user-images.githubusercontent.com/708322/112833648-4b235400-9097-11eb-836d-30b6a37075c2.png)

 -  also, it is not really modal! the task bar still is active, lists spotter as a window, and I can click on other windows
![image](https://user-images.githubusercontent.com/708322/112833618-41015580-9097-11eb-9bc6-87884949bbdd.png)

![image](https://user-images.githubusercontent.com/708322/112833628-4363af80-9097-11eb-92f9-3d226eebc58d.png)

**Comment by guillep** [comment]
Also, while focus is in spotter, global shortcuts are still available (that is ok), so I can open iceberg, or a playground.
What is strange, is that opening iceberg or whatever does not close spotter.

**Comment by estebanlm** [comment]
yeah, this is not a problem of spotter but an issue with how modal works. 
while we can improve is behaviour, I would not call this a bug... and this is not the place for this issue, I will move it to Spec ;)
